### PR TITLE
missed super().__init__() call in concat/adder. Setting ._name instea…

### PR DESCRIPTION
**Pull request by bnb32**
…d of .name protected attribute

### CODE REVIEW

**Comment by grantbuster** [comment]
@bnb32 why not pass layer into super init? I think it's reasonable to name the layer using the intended keras layer arg: https://www.tensorflow.org/api_docs/python/tf/keras/layers/Layer

**Comment by bnb32** [comment]
> @bnb32 why not pass layer into super init? I think it's reasonable to name the layer using the intended keras layer arg: https://www.tensorflow.org/api_docs/python/tf/keras/layers/Layer
> 
> By using ._name we're making this feature confusing and arbitrary with respect to the parent class layer name.

Yeah, makes sense. Was just matching the skip connection layer here but I'll change all the layers to pass through super().

**Comment by grantbuster** [comment]
> > @bnb32 why not pass layer into super init? I think it's reasonable to name the layer using the intended keras layer arg: https://www.tensorflow.org/api_docs/python/tf/keras/layers/Layer
> > By using ._name we're making this feature confusing and arbitrary with respect to the parent class layer name.
> 
> Yeah, makes sense. Was just matching the skip connection layer here but I'll change all the layers to pass through super().

Oh yeah i see. The skip connection layer uses the name internally so the protected attr was fine, but yeah i dont see any reason why we shouldn't just use the parent class name attribute for everything

File: phygnn/layers/custom_layers.py [hunk 1]
```diff
@@ -478,8 +478,7 @@ def __init__(self, name):
             Unique string identifier of the skip connection. The skip endpoint
             should have the same name.
         """
-        super().__init__()
-        self._name = name
+        super().__init__(name=name)
```
Comment (grantbuster) [inline phygnn/layers/custom_layers.py:481]: See the error message on line 509 that needs to be udpated

Comment (bnb32) [reply phygnn/layers/custom_layers.py:481]: the super class initializes a `_name` attribute

Comment (grantbuster) [reply phygnn/layers/custom_layers.py:481]: Ah gotcha, haha so _name was basically the same as name huh? Well all good then. I do think it's better to use .name everywhere then but all good, doesnt matter

**Review by grantbuster — APPROVED** [review]

**Pull request merged**